impeller/compiler/compiler_unittests.cc

```diff
@@ -126,8 +112,7 @@ TEST_P(CompilerTest, UniformsHaveBindingAndSet) {
       suite_name, CompilerTest,                                           \
       ::testing::Values(                                                  \
           TargetPlatform::kOpenGLES, TargetPlatform::kOpenGLDesktop,      \
-          TargetPlatform::kMetalDesktop, TargetPlatform::kMetalIOS,       \
-          TargetPlatform::kFlutterSPIRV),                                 \
+          TargetPlatform::kMetalDesktop, TargetPlatform::kMetalIOS),      \
```

Comment:
I just spotted this. Why not parameterize SKSL generation in the tests?

Reply:
I played with this. The SkSL backend only works with fragment shaders, so the set of tests will be disjoint from the tests for other backends. The SkSL backend is tested by the `dart:ui` unit tests and in the framework repo, but I can add a quick sanity check here in a follow-up PR.